pages/exercises/[lessonSlug].tsx

```diff
@@ -91,21 +91,30 @@ const Exercises: React.FC<QueryDataProps<GetExercisesQuery>> = ({
   )
 }
 
+type ExerciseData = {
+  problem: string
+  answer: string
+  explanation: string
+}
+
 type ExerciseProps = {
-  exercise: ExerciseCardProps
+  exercise: ExerciseData
   setExerciseIndex: React.Dispatch<React.SetStateAction<number>>
   lessonTitle: string
-  showPreviousButton: boolean
-  showSkipButton: boolean
+  hasPrevious: boolean
+  hasNext: boolean
 }
 
 const Exercise = ({
   exercise,
   setExerciseIndex,
   lessonTitle,
-  showPreviousButton,
-  showSkipButton
+  hasPrevious,
+  hasNext
 }: ExerciseProps) => {
+  const [answerShown, setAnswerShown] = useState(false)
+  const [messageKey, setMessageKey] = useState<MessageKey>('EMPTY')
```

Comment:
Do you think it's better to export the `enum` and use it here like `useState(Message.EMPTY)`?

Reply:
Good idea! Thanks for the suggestion.